test/util/collapse-icon.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html>
+<head>
+  <meta name='viewport' content='width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes'>
+  <title>hide-content-tests</title>
+
+  <script src='../../src/bower_components/webcomponentsjs/webcomponents-lite.min.js'></script>
+  <script src='../../src/bower_components/web-component-tester/browser.js'></script>
+  <script src='../../src/bower_components/iron-test-helpers/mock-interactions.js'></script>
+
+  <!-- Import the element to test -->
+  <link rel='import' href='../../src/util/hide-content.html'>
+
+</head>
+<body>
+  <test-fixture id='basic'>
+    <template>
+      <hide-content></hide-content>
+    </template>
+  </test-fixture>
+
+  <script>
+    suite('<hide-content>', function() {
+      var elem;
+
+      setup(function() {
+        elem = fixture('basic');
+      });
+
+      test('tapping hide comments hides content', function() {
```

Comment:
copy pasta mistake

'tapping hides content' not 'tapping hide comments hides content'